### PR TITLE
Support pickups with multiple original locations

### PR DESCRIPTION
**Pull request by henriquegemignani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Dread
 
+- Added: Progressives pickups can now be configured to be placed vanilla.
```
Comment (ThanatosGit) [inline CHANGELOG.md:82]: It's `Progressive pickups` and not `ProgessiveS pickups`, isn't it?

**Review by ThanatosGit — CHANGES_REQUESTED** [review]
lgtm. Can be implemented later for the other games.

**Comment by Miepee** [comment]
Do you want to get that in before the june release?

**Comment by toasterparty** [comment]
Does this mean you could put all missiles, power bomb expansions, etanks, etc. to their vanilla locations? I feel like that would finally end the confusion as to what exactly "major/minor" means by letting the user configure it themselves instead of checking a box.

**Comment by duncathan** [comment]
> I feel like that would finally end the confusion as to what exactly "major/minor" means by letting the user configure it themselves instead of checking a box.

I'm not sure what you mean here?

**Comment by henriquegemignani** [comment]
> Does this mean you could put all missiles, power bomb expansions, etanks, etc. to their vanilla locations? I feel like that would finally end the confusion as to what exactly "major/minor" means by letting the user configure it themselves instead of checking a box.

AmmoPickupDefinition don't support this actually. And the later part has nothing to do with this feature.

**Comment by toasterparty** [comment]
What I'm saying is if `AmmoPickupDefinition` did support this, then you can replace the major/minor functionality with this because they are functionally equivalent.

**Comment by henriquegemignani** [comment]
No? This is just assigning certain pickups to an specific location when creating the pool. It doesn't even touch the generator, like major/minor does.

**Comment by duncathan** [comment]
> What I'm saying is if `AmmoPickupDefinition` did support this, then you can replace the major/minor functionality with this because they are functionally equivalent.

I still don't follow. What exactly would be replaced? And for what purpose? I'm confused what this PR is supposed to have to do with major/minor

File: CHANGELOG.md (hunk 1 above)
Comment (ThanatosGit) [reply CHANGELOG.md:82]: It's just the changelog and ready to go?

**Review by ThanatosGit — APPROVED** [review]

**Pull request merged**